src/index.html

```diff
@@ -155,58 +148,50 @@ <h3><a class="anchor" id="marker-specification"></a>Markers</h3>
           </tr>
         </tbody>
       </table>
-      <p>
-        The markers can be printed on a black-and-white printer.
-      </p>
       <p>
         Sixteen arena wall markers are positioned around the arena as shown in the <a href="#arena-specification">arena specification</a>. Marker codes are mirrored on each side of the arena.
       </p>
       <p>
-        Wall markers are placed such that the bottom of the marker is 20 ±5mm from the arena floor.
+        Wall markers are placed such that the bottom of the marker is 20±5mm from the arena floor.
       </p>
       <h3><a class="anchor" id="arena-specification"></a>Arena</h3>
       <img src="./fig-arena.svg" alt="Arena diagram" width="90%" height="auto">
       <ol>
         <li>
-          The above figure represents, to scale, a birds eye view of the arena. The arena is
-          <strong>5.4m</strong> square.
+          The above figure represents, to scale, a birds eye view of the arena. The arena is a square with side length <strong>5.4m</strong>.
         <li>
           The floor of the arena is covered with a closed-loop, short pile carpet.
         <li>
-          The perimeter of the arena floor is delimited by the arena wall, which has a minimum height of 100mm.
-        <li>
-          Starting areas are 1000 ±50mm squares and will be marked by tape which is considered part of the starting area.
+          The perimeter of the arena floor is delimited by the arena wall, which has a minimum height of 250mm.
         <li>
-          Starting area 1 is at the top of the arena diagram, and area 2 is at the bottom.
+          The inner walls of the arena have a minimum height of 170mm.
         <li>
-          The green boxes denote a robot in their start locations, shown in yellow. Robots may start anywhere inside their starting area facing any direction.
+          Scoring lines are marked using tape with width 50±5mm.
         <li>
-          The inner walls of the arena have a minimum height of 170mm.
+          Starting areas are 1000±50mm squares and will be marked by tape which is considered part of the starting area. Robots may start anywhere inside their starting area facing any direction.
         <li>
-          SourceBots reserves the right to have match officials in the arena during games.
+          SRO reserves the right to have match officials in the arena during games.
       </ol>
       <h3><a class="anchor" id="flag-specification"></a>Robot flags</h3>
       <img src="./fig-flag.svg" alt="Robot flag" width="40%" height="auto">
```

Comment:
This might be better served by longer alt text?

```suggestion
      <img src="./fig-flag.svg" alt="A small red flag, attached to a flag pole" width="40%" height="auto">
```